### ISSUE TITLE
[uni-pagination] Observe only API attributes

### ISSUE THREAD

**Issue by tomalec** [Starcounter/uniform.css#164]
At post-review of https://github.com/Starcounter/uniform.css/pull/137/files I noticed we may consider narrowing down the number of `._refresh()` and mutation observer callback calls.

We can use [`MutationObserverInit#attributeFilter`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit/attributeFilter) not to react on changes of `class`, `id` or `foo` attributes.

//cc @alshakero

**Comment by alshakero** [comment]
Good idea in principle. But I don't see value in it. Because:

1. The chance of these attribs changing is near zero.
2. If they somehow do, the cost of refeshing is trivial and happens once; it's not an animation or a scroll side effect. It's a single operation. And in my opinion most single operations outside a loop or an animation need to be very very very slow to be considered. I mean even 100ms of delay is ok here, and my estimation is that it takes 2ms to refresh.

**Comment by tomalec** [comment]
I totally agree it is not super important, and may not give significant performance benefit, I have no numbers.

But speaking of chance, I think there is a chance of anybody setting any attribute on `uni-pagination` is,  for example, ShadyDOM adds those `class="style-scope host-name"` thingy.
Plus `refresh` is 10% of the codebase of the element so I thought it does something ;)

Putting aside performance it's also maintenance optimization, imagine debugging `_refresh` being hit when somebody else changes `onclick`,`tabindex`, `style`, or some `aria-` attribute
